### PR TITLE
Add generic dedup playbook

### PR DESCRIPTION
**Pull request by erezh31**
improve dedup scripts

### CODE REVIEW

File: Scripts/script-FindSimilarIncidentsByText.yml [hunk 1]
```diff
@@ -8,16 +8,16 @@ script: |-
   TEXT_FIELDS = set(map(lambda x: x.lower(), demisto.args()['textFields'].split(',')))
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidentsByText.yml]: all the changes in this file won't break backward?

File: Scripts/script-FindSimilarIncidents.yml [hunk 2]
```diff
@@ -108,7 +108,7 @@ script: |-
   def get_incidents_by_keys(similar_incident_keys, incident_time, incident_id, hours_back, ignore_closed, max_number_of_results):
       similar_keys_query = " and ".join(map(lambda t: '%s:"%s"' % (t[0], t[1]), similar_incident_keys.items()))
       incident_time = parse_datetime(incident_time)
-      max_date = incident_time
+      max_date = incident_time + timedelta(hours=hours_back)
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidents.yml]: all the changes in this file won't break backward?

Comment (erezh31) [reply Scripts/script-FindSimilarIncidents.yml]: No.. it's just expand the query to search across more time (like the other scripts)

File: Scripts/script-FindSimilarIncidentsByText.yml (hunk 1 above)
Comment (erezh31) [reply Scripts/script-FindSimilarIncidentsByText.yml]: no.. changed const in the code to arguments with the same default value as the consts

**Pull request merged**